austrakka/utils/options.py

```diff
@@ -284,6 +285,21 @@ def opt_read(**attrs: t.Any):
     )
 
 
+def opt_upload_mode(**attrs: t.Any):
+    defaults = {
+        'help': 'Overwrite or Skip if existing sequences are found. '
+                'If omitted, you will get an error.',
+        'required': False,
+    }
+    return _create_option(
+        "-um",
+        '--upload-mode',
+        'mode',
+        type=click.Choice(MODES),
```

Comment:
Can I request we turn this into two mutually exclusive binary flags? `--skip` and `--overwrite` (or possibly --force but I think overwrite is more specific)

Happy to make a PR to the PR

Reply:
Added --skip and --force